queen_attack/queen_attack.rb

```diff
@@ -0,0 +1,65 @@
+class Queens
+	def initialize(white = nil, black = nil)
+		begin
+	    @white = white
+	    @black = black
+	  raise ArgumentError, "You don't enter a position." if correct_value?	 
```

Comment:
Usually better to use if/else instead of raise error. (Google why and tell me on interview)